docs/devel/integration.rst

```diff
@@ -55,17 +58,22 @@ translations to match your code base.
 Adding new strings
 ++++++++++++++++++
 
-In case your translation files are stored in VCS together with the code, you most
-likely have existing workflow for developers to introduce new strings. You
-might extend it by using :ref:`source-quality-gateway`.
+In case your translation files are stored in VCS together with the code,
+you most likely have existing workflow for developers to introduce new strings.
+You might extend it by using 
+
+Any way of adding strings will be picked up, but consider using
+:ref:`source-quality-gateway` to avoid also introducing errors.
 
-When the translation files are separate, there needs to be a way to introduce
-new strings. Weblate can add new strings on monolingual translations only (see
-:ref:`bimono`). You have three options to do that:
+When the translation files are not in a upstream repository, there needs to be a way to introduce
+new strings.
```

Comment:
The link to bi/mono-lingual format is gone...

Reply:
It is a bit farther down. I seem to have thought there are different rules for the API…

Reply:
It is not related to the API, but is general Weblate limitation (for now).

Reply:
I rewrote this again as all three options are monolingual only.